### PR TITLE
Small fixes + workflow improvement

### PR DESCRIPTION
**Pull request by isinyaaa**
- config: remove unneded package specifications on `cargo bin --install`
- README: fix typo `cargo-bin-run` -> `cargo-run-bin`
- Add alias for `cargo-make`

### CODE REVIEW

**Comment by isinyaaa** [comment]
Updated to improve commit descriptions.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ A website for learning and interacting with ultrasound images.
...
 This will install the components and targets used in this project.
-The `cargo-bin-run` binary will be installed globally and will make sure that you
-have the correct version of each binary dependency.
-To work with binaries such as `trunk`, use `cargo bin trunk` in order to point to
-the correct version of `trunk` installed locally (at `.bin/`).
+The [`cargo-run-bin`](https://github.com/dustinblackman/cargo-run-bin) binary
+will be installed globally and will make sure that you have the correct version
+of each binary dependency.
+To work with binaries such as `trunk`, use `cargo bin trunk` in order to point
+to the correct version of `trunk` installed locally (at `.bin/`).
```
Comment (luizmugnaini) [inline README.md:15]: If the above suggestion is accepted, we could refer to the fact that we have an alias `trunk` for `cargo bin trunk`. We could also mention the `make` alias.

File: .cargo/config.toml
Comment (luizmugnaini) [inline .cargo/config.toml:1]: I think we could add an alias for `trunk`, like
```
trunk = "bin trunk"
```

**Review by luizmugnaini — CHANGES_REQUESTED** [review]

File: .cargo/config.toml
Comment (isinyaaa) [reply .cargo/config.toml:1]: Ohh that's a good idea. will do

File: README.md (hunk 1 above)
Comment (isinyaaa) [reply README.md:15]: oops I missed this, will update now.

Comment (isinyaaa) [reply README.md]: is this version okay?

Comment (isinyaaa) [reply README.md]: I think I will add another line to mention `make`.

**Review by luizmugnaini — APPROVED** [review]

**Pull request merged**